.github/workflows/applicationTesting.yml

```diff
@@ -32,3 +32,10 @@ jobs:
     - name: Run Tests
       run: |
         pytest
+    
+    - name: Code Coverage Checking
+      uses: tadomikikuto-bit/django-coveragepy-github-action@master
```

Comment:
waouh.....
Do you create your own Github action for this?
it's really really nice... I like it...

Reply:
I customized a forked action to suite our needs